docker/Dockerfile

```diff
@@ -12,13 +12,6 @@ FROM nvidia/cuda:${CUDA_VERSION}-devel-ubuntu18.04 as base
 # Source: https://github.com/NVIDIA/nvidia-docker/issues/1632
 
 ARG VENV
-
-# metainformation
-LABEL org.opencontainers.image.version = "1.4"
-LABEL org.opencontainers.image.authors = "Neuralmagic, Inc."
-LABEL org.opencontainers.image.source = "https://github.com/neuralmagic/sparseml"
-LABEL org.opencontainers.image.licenses = "Apache License 2.0"
```

Comment:
Do we not need this anymore?

Reply:
No this was added by an external user, it's good but adds one more thing we'll have to maintain, the tags pushed to ghcr already take care of the image version